app/services/proofing/resolution/progressive_proofer.rb

```diff
@@ -202,6 +201,22 @@ def threatmetrix_disabled_result
         )
       end
 
+      def threatmetrix_pii_missing_result
+        Proofing::DdpResult.new(
+          success: false,
+          client: 'tmx_pii_missing',
+          review_status: 'fail',
+        )
+      end
+
+      def threatmetrix_id_missing_result
+        Proofing::DdpResult.new(
+          success: false,
+          client: 'tmx_id_missing',
```

Comment:
Nit: can this be `tmx_session_id_missing`?

Reply:
Done.